markdown/org/docs/patterns/charlie/instructions/en.md

```diff
@@ -215,78 +226,94 @@ Getting it just right is one of those things you'll end up cherishing each time
+Trim the fly facing to half of the seam allowance.	
+Tuck in the tip of the fly facing and secure it with a few stitches.
+Edge stitch the fly facing to the seam allowance.
+Press the seam allowance toward the facing and understitch the facing for a clean finish. Again, keeping the right leg 	seam allowance out of the way.
 
-Sew the fly facing to the seam allowance. Make sure to leave the front panel out of it.
+![Finish Facing](FinishFacing.svg)
 
-### Press cross seam, fly facing and the fly edge
+### Edge stitch the zipper to the fly extension
 
-Press open the cross seam. Press the fly facing to the side.
+<tip>
 
-Then, fold over the extra seam allowance on the right front leg and press it down.
+**Zipper Length**
 
-### Baste the zipper to the fly extension
+It's a good idea to ensure your zipper stop ends before the indicated stitch line on the pattern. This will ensure you don't sew over your zipper stop and reduces tension on the bottom of the fly. If your zipper is too long, you can do a 0 length zig zag stitch to create a new stop (on just the zipper itself).
 
-Place the zipper along the (slightly curved) edge of the fly extension.
+</tip>
 
-Now sew along the edge. No need to get close to the zipper, just sew along the edge to hold them in place.
+Edge stitch along the left edge of the zipper to the serged edge of the fly extension to hold it in place.
 
-### Attach zipper to the fly
+![Baste Zipper](BasteZipper.svg)
 
-Now place the right front leg's fly with the extra seam allowance that you folded back on top of the zip.
 
-Use a zipper foot edge-stitch the folded-over edge along the zipper edge.
+### Top stitch the right leg to the zipper
 
-<Tip>
+Start by pressing the right leg seam allowance to the back.
 
-You might want to baste this first to make sure it is nicely parallel to the zipper. Close, but not too close.
+<tip>
 
-</Tip>
+Remember that the seam allowance starts slightly right of the center front, the center front is marked in purple in the image, make sure you don't fold along the center front line by mistake.
+
+</tip>
 
-### Sew the zipper to the fly facing
+Ensure you place the folded edge of the right leg just left of the zipper teeth to prevent fabric from being caught in the zipper.
 
-Now sew the other side of the zipper to the fly facing. Make sure to align it properly so that
-when the trousers are closed, the zip is hidden.
+Stop sewing at the top 'fly extention' notch, slightly above the cross seam, and be sure to backstitch. If you don't do this you won't be able to keep the extension out of the way when sewing the J-Seam.
 
-### Finish the cross seam
+![Attach Extension to Right Leg](AttachExtensionToRightLeg.svg)
+
+
+### Attach the zipper to the fly facing.
+
+Be careful to align the leg left on the right center front. It can be valuable to pin or baste through the facing and right zide of the zipper tape. (We need to sew the left side of the zipper tape to the left leg).
```

Comment:
```suggestion
Make sure to align the leg left to the right center front. It can be valuable to pin or baste through the facing and _good side_ of the zipper tape. (We need to sew the left side of the zipper tape to the left leg).
```

Reply:
good side isn't the intended meaning actually, I mean the zippers 'right side' like, right leg.